src/datepicker/__tests__/datepicker-keyboard-nav.e2e.js

```diff
@@ -107,4 +114,165 @@ describe('Datepicker - keyboard navigation', () => {
     isElActive = await isActiveEl(page, selectors.apr2);
     expect(isElActive).toBe(true);
   });
+
+  it('calendar sets highlighted date appropriately when selecting a new date or navigating around', async () => {
```

Comment:
can we break this test case into a few smaller ones?

Reply:
I want to test the whole chain vs separate actions to make sure that, for example, selecting a date doesn't affect the rest of functionality.